queries.sql

```diff
@@ -213,8 +213,31 @@ ORDER BY COUNT(species.name) DESC
 LIMIT 1;
 
 
+------------------------------------------------------------------------------------------------------------------------------
+  -- performance audit
+------------------------------------------------------------------------------------------------------------------------------
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4
+-- the above query returns an execution time of 2757.049 ms
+CREATE INDEX visits_animal_id_idx ON visits (animal_id);
```

Comment:
- [ ] @ team great work on your queries, however, the indexes creation should be done in the `schema.sql` file, kindy move them to your schema file, your queries file should contain only queries to your database.

Reply:
Thanks for the feedback. On it!